packages/samples/mvp-dapp/src/chain-source.ts

```diff
@@ -13,7 +13,7 @@ export class NervosChainSource implements ChainSource {
 
   getCurrentEpoch = () => this.#rpc.getCurrentEpoch()
 
-  getBlock = (blockNumber: string) => this.#rpc.getBlock(blockNumber)
+  getBlock = (blockNumber: string) => this.#rpc.getBlockByNumber(blockNumber)
```

Comment:
Here the use of `getBlock` will thorw an error, so changed to `getBlockByNumber`